### ISSUE TITLE
LTS, Important: SecurityToken field mandatoryness

### ISSUE THREAD

**Issue by milux** [IndustrialDataSpace/InformationModel#444]
Currently, the SecurityToken/DAT is defined as a mandatory field for all IDS message classes.
This perfectly makes sense for MIME/Multipart and IDS-LDP due to statelessness, but does **not** make sense for IDSCP2.
IDSCP2 handles the DAT (Dynamic Attribute Token) within the protocol itself, and the transfer of it in the serialized message headers creates unnecessary overhead and makes the protocol unnecessarily complicated.

Could we make this field optional?

**Comment by milux** [comment]
Please assign @milux and @obraunsdorf to this issue, and add tags LTS and High Prio or something, since I cannot do that. Thank you!

**Comment by JohannesLipp** [comment]
@milux @obraunsdorf is there any update on this? Thank you!

**Comment by milux** [comment]
> @milux @obraunsdorf is there any update on this? Thank you!

Not really, Sebastian asked me to open that issue, we are still awaiting a reply.
